hikari/impl/bot.py

```diff
@@ -207,6 +207,9 @@ class GatewayBot(traits.GatewayBotAware):
         Defaults to `hikari.intents.Intents.ALL_UNPRIVILEGED`. This allows you
         to change which intents your application will use on the gateway. This
         can be used to control and change the types of events you will receive.
+    chunk_members: builtins.bool
+        Defaults to `builtins.True`. If `builtins.False`, then no member chunks
+        will be requested automatically, even if the members intent is enabled.
```

Comment:
You can run the pages pipeline and check the output there. 

Best way to do this would be to execute `nox pages test-pages`, which will generate them and then spin up an HTTP server so you can view them

Reply:
https://circuit.is-from.space/snap_05676f9f.png